how-to/openldap/install-openldap.md

```diff
@@ -262,37 +303,39 @@ olcDbIndex: mail eq,sub
 ### Change the RootDN password:
 
 First, run `slappasswd` to get the hash for the new password you want:
-
 ```bash
-$ slappasswd
-
+slappasswd
+```
+and enter and confirm the new password
+```text
 New password:
 Re-enter new password:
 {SSHA}VKrYMxlSKhONGRpC6rnASKNmXG2xHXFo
 ```
 
-Now prepare a `changerootpw.ldif` file with this content:
-
+Now prepare a `changerootpw.ldif` file with this content (replacing the last line with the hash you got from the previous command):
 ```text
 dn: olcDatabase={1}mdb,cn=config
 changetype: modify
 replace: olcRootPW
-olcRootPW: {SSHA}VKrYMxlSKhONGRpC6rnASKNmXG2xHXFo
+olcRootPW: {SSHA}VKrYMxlSKhONGRpC6rnASKNmXG2xHXFo 
 ```
 
 Finally, run the `ldapmodify` command:
-
 ```bash
 $ sudo ldapmodify -Q -Y EXTERNAL -H ldapi:/// -f changerootpw.ldif
```

Comment:
While fixing the commands to be able to be simple single lines for copy and paste - could we also fix the three remaining ones with leading $? This is one of them.